feature-libs/product-configurator/rulebased/components/attribute/types/multi-selection-bundle/configurator-attribute-multi-selection-bundle.component.ts

```diff
@@ -61,11 +63,31 @@ export class ConfiguratorAttributeMultiSelectionBundleComponent
     };
 
     const event: ConfigFormUpdateEvent = {
-      ownerKey: this.ownerKey,
       changedAttribute: {
         ...this.attribute,
         values: this.multipleSelectionValues,
       },
+      ownerKey: this.ownerKey,
+      updateType: Configurator.UpdateType.ATTRIBUTE,
+    };
+
+    return event;
+  }
+
+  protected updateMultipleSelectionValuesQuantity(eventValue) {
+    const value: Configurator.Value = this.multipleSelectionValues.find(
+      (value) => value.valueCode === eventValue.valueCode
+    );
+
+    value.quantity = eventValue.quantity;
```

Comment:
what if `value` is `undefined`? (above `.find()` function doesn't find any value). is that a case or will `.find()` always have value?

Reply:
true, so maybe add if (!value) return; where no value function return and nothing happen